docs/workflows/README.md

```diff
@@ -0,0 +1,201 @@
+![tooling-validate-light](tooling-validate-light.png#gh-light-mode-only)
+
+**Jobs**
+
+- **Config**: Checking the input parameters and generating various configuration properties. Creating a summary report from the configuration properties.
+
+- **Security**: Basic security and vulnerability checks using the [gosec](https://github.com/securego/gosec) and [govulncheck](https://pkg.go.dev/golang.org/x/vuln/cmd/govulncheck) tools. If the check fails, the workflow will stop with an error.
+
+- **DevContainer**: If a [Development Container](https://containers.dev/) configuration exists, it validates its parameters based on the workflow input parameters. In case of incorrect configuration, it indicates an error but does not stop the workflow.
+
+- **Lint**: Static analysis of the source code using the [golangci-lint](https://golangci-lint.run/) tool.  It uses the golangci-lint configuration found in the repository, or if it is missing, the following arguments:
+  ```
+  --no-config --presets bugs --enable gofmt
+  ```
+  In case of an error, the workflow will stop with an error.
+
+- **Smoke**: Run short Go tests (`-short` flag) on single platform and single go version. In case of an error, the workflow will stop with an error. The role of this job is to quickly stop the workflow in case of trivial test errors, preventing slower tests from running on multiple platforms using multiple go versions.
+
+- **Test**: Run Go tests on multiple platforms using multiple go versions. The execution will be done using a matrix strategy, the number of jobs run is the number of go versions multiplied by the number of platforms. The workflow run does not stop if the first job fails, all jobs in the matrix are executed. If any job fails, the workflow will stop with an error. A typical input contains two go versions and three platforms (`ubuntu-latest`, `windows-latest`, `macos-latest`) which is 6 jobs in the matrix.
+
+- **Build**: Create builds using [GoReleaser](https://goreleaser.com/) tool on multiple platforms using multiple go versions. The build will be done with a matrix strategy, the number of jobs run is the number of go versions multiplied by the number of platforms. The build does not stop on the first job failure, all jobs in the matrix are executed. If any job fails, the workflow will stop. The typical input contains two go versions and three platforms (`ubuntu-latest`, `windows-latest`, `macos-latest`) which is 6 jobs in the matrix.
+  The build is done with the following GoReleaser arguments:
+  ```yaml file=../../.github/workflows/tooling-validate.yml region=build-args
+            args: build --clean --snapshot --single-target
+  ```
+
+  [Bats](https://github.com/bats-core/bats-core) (Bash Automated Testing System) scripts can be specified as integration tests. These will be executed in jobs running on `Linux` operating systems (with all go versions specified in the input). If any integration test script fails, the workflow will stop with an error.
```

Comment:
Shouldn't this have a bullet point also?

Reply:
no, it is just continuation of prev point